drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -2320,6 +2320,11 @@ matterGeneric:
     deviceTypes:
       - id: 0x010B # Dimmable Plug-in Unit
     deviceProfileName: plug-level
+  - id: "matter/electrical-sensor"
+    deviceLabel: Matter Electrical Sensor
+    deviceTypes:
+      - id: 0x0510 # Electrical Sensor
+    deviceProfileName: power-energy-powerConsumption
```

Comment:
@nickolas-deboom we will want to add a default fingerprint for the Valve device type as well. You can just have it match to the `water-valve` profile for now, and the logic you have in the `init` function should handle adding the `level` capability as needed

Reply:
Added a default fingerprint for water valve!